VIDEO_1_SCRIPT.md

```diff
@@ -0,0 +1,99 @@
+
+And while the social platforms were experimenting in the promising world of APIs, Google was going one step further and preparing the way to integrate its tools into third-party sites: just six months after the launch of Google Maps, the Mountain View company launched the mapping service API. They did so in response to the countless fake platforms that emerged based on hacking information from Google's tool.
+
+After the success of its first attempt, Amazon took its APIs to the next level. Far from focusing exclusively on e-commerce, the company opted to create an entire infrastructure to support developers based on REST APIs: first with Amazon S3, to store data in the cloud, and later with Amazon EC2, to have virtual servers in data centers.
+
+Since 2010, the prevalence of the public API has skyrocketed. Mobiles finally brought APIs to our daily lives, every day, every hour, making life easier for every user that was coming and going. We now had support understanding the world with the help of machines that communicated with each other fluently. All this information is within grasp to men and women, merely one-click away. 
+
+E-commerce, Social media, cloud computing, mobility… Without a doubt, we are definitely living in the Web API era.
+
+## Types of APIs
+
+There are 4 types of APIs: 
+
+- **Web services APIs**: these are the application development interfaces that enable the exchange of information between an application and a web service. (A web service is a software that provides access to a particular service via a URL). 
+
+    There are four types of Web services APIs:
+
+    •	SOAP (Simple Object Access Protocol), a standard protocol for exchanging information and data in XML between two objects.
+
+    •	XML-RPC, a remote procedure call protocol that uses XML as the data format and HTTP calls as the communication system.
+
+    • JSON-RPC, the same protocol as our previously explained XML-RPC, but in JSON format. 
+
+    •	REST (Representational State Transfer), a software architecture for hypermedia systems on the World Wide Web; a REST API uses the HTTP protocol.
+
+- **Library-based APIs**: these types of APIs are those that allow an application to import a library from other software, in order to exchange information. Nowadays, many of the libraries that provide access to products and services are designed in JavaScript.
+
+- **Class-based APIs**: this type of application development interface allows the connection with data around classes, as is usual in object-oriented programming with Java. The Java API uses abstract classes for the creation of applications just like any program developed in this language. These classes provide everything needed to perform all kinds of functions within these applications. The Java development interface is organized into packages and each of those packages, in turn, contains a set of related classes.  
```

Comment:
is this really how people talk about this? for me it is just another type of library API